gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -37,6 +40,7 @@ def main(args):
     max_af = args.max_af
     partitions = args.partitions
     use_pops = args.use_pop
+    weighted = args.use_weights
```

Comment:
```suggestion
   use_weights = args.use_weights
```
for consistency with use_pops

Reply:
also, a minor comment (I can't comment above): probably helpful to add "LoF" to the description for this script ("Execute loss-of-function (LoF) constraint pipeline.")